src/main/java/reactor/netty/udp/UdpClient.java

```diff
@@ -136,7 +149,9 @@ public final UdpClient remoteAddress(Supplier<? extends SocketAddress> remoteAdd
 	 * enriched bootstrap.
 	 *
 	 * @return a new {@link UdpClient}
+	 * @deprecated  as of 0.9.10. Use the methods exposed on {@link UdpClient} level.
```

Comment:
same

Reply:
This method is removed in master, so I'll add 1.0.0

Reply:
done